utils/update_checkout/update-checkout-config.json

```diff
@@ -120,7 +122,8 @@
                 "swift-markdown": "main",
                 "swift-cmark-gfm": "gfm",
                 "swift-nio": "2.31.2",
-                "swift-nio-ssl": "2.15.0"
+                "swift-nio-ssl": "2.15.0",
+                "swift-experimental-string-processing": "dev/1"
```

Comment:
Why are we using dev/1 branch? Also this will need to be added to rebranch and next scheme.

Reply:
We'll integrate swift-experimental-string-processing into apple/swift periodically by updating tag. The tagging scheme we chose is `dev/1`, `dev/2`, etc. Do you have a concern about this tagging scheme?

Reply:
I propose that we use `dev/1` for all three schemes ('main', 'rebranch', and 'next'), and update all three tags whenever we integrate. WDYT?